topic-operator/src/main/java/io/strimzi/operator/topic/TopicOperatorUtil.java

```diff
@@ -30,6 +32,18 @@ public class TopicOperatorUtil {
 
     private TopicOperatorUtil() { }
 
+    /**
+     * Create e new instance of Kubernetes client.
+     *
+     * @return Kubernetes client.
+     */
+    public static KubernetesClient createKubernetesClient() {
+        return new OperatorKubernetesClientBuilder(
+            "strimzi-topic-operator-" + System.nanoTime(),
```

Comment:
Why are you using `nanoTime()` here?

Reply:
I guess the original intent was to provide a unique identifier, but it's better to use UUID for that.